src/main/resources/snippets/jython/dispatch.py

```diff
@@ -0,0 +1,5 @@
+def dispatch(data):
+    result = ""
+    for item in data:
+        result = result + item.__str__()
```

Comment:
It indeed delegates to `__str__`. My remark was more about the “pythonish” aspect :)